host/tests/deepsleepClasses/power_testConfig.yml

```diff
@@ -0,0 +1,23 @@
+# this file is generated by the testing suites, when a new test is modified, this configuration file will be filled out and updated.
+
+power:  # Prefix must always exist
+    description: "power Manager testing profile / menu system for UT"
+    test:
+        execute: "" #Execute command
```

Comment:
Add artifacts folder and update execute field (refer dsAudio)

Reply:
Added as requested